src/components/badges/__tests__/buyNow.test.tsx

```diff
@@ -2,7 +2,7 @@ import React from "react"
 import { fireEvent, render, screen } from "@testing-library/react"
 
 import BuyNowBadge from "../buyNow"
-import { DummyTooltip } from "../../../stories/badges/dummyTooltip"
+import { DummyTooltip } from "../../../../.storybook/stories/badges/dummyTooltip"
```

Comment:
I would've not expected to import from stories in tests, looks very weird. 
Can we extract those `dummy` components somewhere where they are properly shared? 
(not necessarily now, but in a follow-up tackled in due time)

Reply:
absolutely - had the same reaction. I already touched so many things in this PR and would like to push that to a separate effort